### ISSUE TITLE
What dll exactly to use?

### ISSUE THREAD

**Issue by evlo** [valinet/ExplorerPatcher#36]
I tried to copy DLL from startAllBack 2.90 RC to **%appdata%\ExplorerPatcher**, but explorer crashes after that, can you please share exact dll for download somethere?

Thank you

It is really minor thing as I'm happy with how your solution works without the new icons.

**Comment by IsraelNS902** [comment]
@evlo 

It's the DLL file included in their TBD (Beta) version. 

It appears that the developers have pulled the plug on the beta as https://startisback.com/tbd seems to redirect to their homepage.

Luckily I've still got a copy of it I can share!

**Download Link:** https://bit.ly/2ZSNQl7

**Comment by valinet** [comment]
Indeed, you basically need to source a DLL called `StartIsBack64.dll` of size 730.016 bytes and with SHA256: A9F95CE4ECEDF299703E92CCBF0F7835F7BCFEC90D95090DCE4735D860C15DCE.

I will work on adding compatibility with newer versions eventually, especially when I find a better strategy of using things from the DLL. I am open to suggestions.

I prefer this approach as I don't want all the things StartIsBack does to the system. With this, I can choose what I want. I'd strongly advise paying for a StartIsBack license if you plan on using it inside this application. Or, you can help port over the features ExplorerPatcher uses from that so the use of it is not needed anymore.
